src/main/java/FrameworksDrivers/LogInView.java

```diff
@@ -73,33 +69,41 @@ public void sendPaths(Object[] paths){
 
 
     @Override
-    public void actionPerformed(ActionEvent e) {
+    public void updatePage(Object[] info) {
+        //LoginPresenter logPresenter = new LoginPresenter();
```

Comment:
What's the reason for having the presenter here? Wouldn't the presenter just call the udpatePage method? You would only need to create a presenter here if you want to call another use case or something I think.